Et ops 4344 accessibility statement

ed1286 opened the pull request:
added accessibility statement to this journey

ed1286 committed:
-phase one of trying to add the accessibility statement to this journey

ed1286 committed:
-phase one of trying to add the accessibility statement to this journey
-linked up

ed1286 committed:
-phase one of trying to add the accessibility statement to this journey
-linked up
-updated messages to make correct accessibility statement

ed1286 committed:
-phase one of trying to add the accessibility statement to this journey
-linked up
-updated messages to make correct accessibility statement
-got rid of Welsh flag

ed1286 committed:
-phase one of trying to add the accessibility statement to this journey
-linked up
-updated messages to make correct accessibility statement
-got rid of Welsh flag
-added link for reporting access problems

ed1286 committed:
-phase one of trying to add the accessibility statement to this journey
-linked up
-updated messages to make correct accessibility statement
-got rid of Welsh flag
-added link for reporting access problems
-made links open a new tab

andrewrichards1 reviewed (commented):
Just needs a test in the same style as the other pages

ed1286 committed:
-phase one of trying to add the accessibility statement to this journey
-linked up
-updated messages to make correct accessibility statement
-got rid of Welsh flag
-added link for reporting access problems
-made links open a new tab
-test working
